i18n/ads-language-pack-de/package.json

```diff
@@ -0,0 +1,638 @@
+{
+	"name": "ads-language-pack-de",
+	"displayName": "German Language Pack for Azure Data Studio",
+	"description": "Language pack extension for German",
+	"version": "1.26.0",
+	"publisher": "Microsoft",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Microsoft/azuredatastudio"
+	},
+	"license": "SEE SOURCE EULA LICENSE IN LICENSE.txt",
+	"engines": {
+		"vscode": "^1.34.0",
+		"azdata": ">=1.26.0"
+	},
+	"icon": "languagepack.png",
+	"categories": [
+		"Language Packs"
+	],
+	"contributes": {
+		"localizations": [
+			{
+				"languageId": "de",
+				"languageName": "German",
+				"localizedLanguageName": "Deutsch",
+				"translations": [
+					{
+						"id": "Microsoft.admin-tool-ext-win",
+						"path": "./translations/extensions/admin-tool-ext-win.i18n.json"
+					},
+					{
+						"id": "Microsoft.agent",
+						"path": "./translations/extensions/agent.i18n.json"
+					},
+					{
+						"id": "Microsoft.azurecore",
+						"path": "./translations/extensions/agent.i18n.json"
```

Comment:
The path appears to be wrong here and elsewhere in this file, please make sure you double check that all the paths are correct in all the package.json's